packages/camera/camera_web/lib/src/camera_service.dart

```diff
@@ -393,4 +394,68 @@ class CameraService {
         return DeviceOrientation.portraitUp;
     }
   }
+
+  /// Maps a [Uint] to a [CameraImageData].
+  CameraImageData getCameraImageDataFromBytes(
```

Comment:
This method seems a little bit superfluous, it seems its implementation could be inlined where this is used?